packages/dds/tree/src/test/utils.ts

```diff
@@ -426,38 +420,84 @@ export class TestTreeProviderLite {
 				idCompressor: createIdCompressor(sessionId),
 				logger: this.logger,
 			});
-			const tree = this.factory.create(
-				runtime,
-				TestTreeProviderLite.treeId,
-			) as SharedTreeWithConnectionStateSetter;
+			const tree = this.factory.create(runtime, `tree-${i}`) as ISharedTree;
 			const containerRuntime = this.runtimeFactory.createContainerRuntime(runtime);
-			this.containerRuntimes.add(containerRuntime);
+			this.containerRuntimeMap.set(tree.id, containerRuntime);
 			tree.connect({
 				deltaConnection: runtime.createDeltaConnection(),
 				objectStorage: new MockStorage(),
 			});
-			(tree as Mutable<SharedTreeWithConnectionStateSetter>).setConnected = (
-				connectionState: boolean,
-			) => {
-				containerRuntime.connected = connectionState;
-			};
 			t.push(tree);
 		}
 		this.trees = t;
 	}
 
-	public processMessages(count?: number): void {
-		// In TurnBased mode, flush the messages from all the runtimes before processing the messages.
-		// Note that this does not preserve the order in which the messages were sent. To do so, tests should
-		// flush messages from individual runtimes in the order they were created.
-		if (this.flushMode === FlushMode.TurnBased) {
-			this.containerRuntimes.forEach((containerRuntime) => {
+	/**
+	 * To be used in TurnBased mode where messages that are sent are not automatically flushed.
```

Comment:
It would be good to clarify that `flushMessages` has no effect on trees that are disconnected.